src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTxWithDoubleSpendSpecification_Steps.cs

```diff
@@ -95,7 +94,7 @@ private void txn_mempool_conflict_error_occurs()
         private void receiving_node_attempts_to_double_spend_mempool_doesnotaccept()
         {
             var unusedAddress = this.stratisReceiver.FullNode.WalletManager().GetUnusedAddress(new WalletAccountReference("mywallet", "account 0"));
-            var transactionCloned = this.transaction.Clone();
+            var transactionCloned = this.transaction.Clone(this.stratisReceiver.FullNode.Network.Consensus.ConsensusFactory);
```

Comment:
ah here you need a consensusFactory for a transaction it seems.

Reply:
Yeah see comment above.